README.md

```diff
@@ -24,6 +24,8 @@ Where opts is an object with the following properties (all optional):
 * **port** - the desired port for the server to use. Defaults to `8000`.
 * **dir** - the directory to launch from (where it should look for a Cordova project). Defaults to cwd.
 * **simhostui** - the directory containing the UI specific files of the simulation host. Defaults to the bundled simulation host files, found in `src/sim-host/ui`.
+* **nolivereload** - A boolean. Set to true to disable live reload. Defaults to false.
```

Comment:
Instead of nolivereload, we can just name this property livereload, and the default value is true.